src/web/services/instructor.service.ts

```diff
@@ -122,11 +122,11 @@ export class InstructorService {
     }
 
     if (queryParams.instructorRole) {
-      paramMap.sectionname = queryParams.instructorRole;
+      paramMap.instructorrole = queryParams.instructorRole;
     }
 
     if (queryParams.instructorEmail) {
-      paramMap.instructorEmail = queryParams.instructorEmail;
+      paramMap.instructoremail = queryParams.instructorEmail;
```

Comment:
Same here?

Reply:
This change is also due to a typo in the current code. 
Backend expects the param name `Const.ParamNames.INSTRUCTOR_EMAIL` to be the lower case `instructoremail`